zetaclient/chains/bitcoin/tokenizer.go

```diff
@@ -0,0 +1,156 @@
+package bitcoin
+
+import (
+	"encoding/binary"
+	"fmt"
+
+	"github.com/btcsuite/btcd/txscript"
+)
+
+func newScriptTokenizer(script []byte) scriptTokenizer {
```

Comment:
Does `script` have a min or max bytes size that we should be checking here?

Reply:
It's coming from Bitcoin, it should be contained. I think we can just take whatever comes from BTC.

Reply:
Makes sense, I'm not familiar with this scripts so that's why I was wondering if there's a possibility of script being 0 bytes, on which case we can simply discard it and save resources.